### PR TITLE
[redis-ha] fix: use release name as prefix for sentinel svc

### PR DESCRIPTION
**Pull request by tuananh**
I want to create multiple releases for `redis-ha` but it will fail because `sentinel-svc` service is already there.

This PR will add release name as prefix to sentinel-svc. I'm not sure if it will affect but everything seems fine.

@smileisak can you help me confirm this? if it's not ok, can you guide me where i need to change the sentinel svc reference?

### CODE REVIEW

**Comment by viglesiasce** [comment]
ping @smileisak for review

**Review by smileisak — CHANGES_REQUESTED** [review]
@tuananh can you fix conflicts please. i'm also working to add new PR to modify slave deployment to statefulset and add one pvc per profile ( master-pvc and slave-pvc )

File: stable/redis-ha/values.yaml [hunk 1]
```diff
@@ -70,7 +70,6 @@ persistentVolume:
 redis_image: gcr.io/google_containers/redis:v1
 
 
-
```
Comment (smileisak) [inline stable/redis-ha/values.yaml]: Can you delete this line?

**Review by smileisak — CHANGES_REQUESTED** [review]

**Comment by smileisak** [comment]
@tuananh you need to configure [upstream](https://help.github.com/articles/syncing-a-fork/)  fix conflict and re push. :yum:

File: stable/redis-ha/values.yaml (hunk 1 above)
Comment (tuananh) [reply stable/redis-ha/values.yaml]: removed. but circici linter complains about too many blank lines 😕

**Comment by tuananh** [comment]
@smileisak resolved. I still want to ask where the service name is used though :D

File: stable/redis-ha/templates/redis-sentinel-service.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ metadata:
     name: {{ template "name" . }}-sentinel-svc
     role: service
 {{ include "labels.standard" . | indent 4 }}
-  name: redis-sentinel
+  name: {{ .Release.Name }}-redis-sentinel
```
Comment (unguiculus) [inline stable/redis-ha/templates/redis-sentinel-service.yaml:8]: The correct name should use the `fullname` template.

`{{ template "fullname" . }}-redis-sentinel`

For some reason, some of the chart's resources also have name labels which don't match the resource name. Seems weird to me.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
Marking stale. Please update within one week.

**Comment by unguiculus** [comment]
Closing as stale. Feel free to reopen if/when you decide to pick this up again.

**Pull request closed without merging**